### ISSUE TITLE
Multiple player accounts being selected at once. [BUG]

### ISSUE THREAD

**Issue by BlahBlah81** [MultiMC/Launcher#4265]
### Operating System

Linux

### Description of bug

Multiple accounts are selected at once causing the first one to be used and never the second. Image for reference [https://imgur.com/a/Zt42eBA](url)

### Steps to reproduce

Use Dev version
Have 2 accounts
Select one
They are all now selected

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by BlahBlah81** [comment]
There is a temp fix i found for this though.
Click on "no default"
Start modpack to get an account choosing prompt.

**Comment by wafflecoffee** [comment]
Yep, can confirm after updating from 0022aed8bb03c21ca6828092f7f4c5655699f9a9 to c2c56a2f6ceaedb8a3fa88c848b345db0fec7f9c. [Video demonstration](https://youtu.be/Wzn-tzhi5rs)

**Comment by BloodshotPico** [comment]
> There is a temp fix i found for this though. Click on "no default" Start modpack to get an account choosing prompt.

I can vouch for this temp fix. Seems to be working fine using that method for me so far.

**Comment by NotNite** [comment]
Can confirm both the bug and workaround on Windows 10. Some observations:

- Switching to the stable branch fixes the bug. (:P)
- The profile that it shows/logs in with is the last profile in my profile list.
- The workaround doesn't let me set the selected profile as default.
- The workaround needs to be done for every launch.

**Comment by jamierocks** [comment]
Yeah, can confirm I got this bug in my development environment too.

**Comment by Robaschi** [comment]
Got this bug as well, it's even present in the newest develop version on Windows 10. 0.6.13-develop-2892.
The workaround also works on this version too.

**Comment by neclimdul** [comment]
looks like b49987e8767b4d0076d62033b019507a8cbc801c was the fix.
